spec/controllers/solidus_bolt/webhooks_controller_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe 'SolidusBolt::WebhooksController', type: :request do
+  describe '#update' do
+    subject(:endpoint_call) { post "/webhooks/bolt", params: {} }
+
+    context "when valid" do
+      let(:expected_params) { ActionController::Parameters.new({}) }
+      let(:hash_header)     { nil }
+
+      before do
+        allow(::SolidusBolt::Webhooks::Sorter).to receive(:call)
+        endpoint_call
+        expected_params[:controller] = "solidus_bolt/webhooks"
+        expected_params[:action] = "update"
```

Comment:
Why did you set these in the before hook, instead of in the dedicated let at line 10?